services/src/main/java/uk/gov/moj/sdt/services/SubmitQueryService.java

```diff
@@ -433,23 +433,21 @@ private void sendRequestToTargetApp(final ISubmitQueryRequest submitQueryRequest
         SubmitQueryResponse mcolSubmitQueryResponse =
             requestConsumer.submitQuery(submitQueryRequest, connectionTimeOut, requestTimeOut);
 
-        if (Boolean.TRUE.equals(submitQueryRequest.getBulkCustomer().getReadyForAlternateService())) {
```

Comment:
Should this have been removed?  Do we always want to query CMC regardless of whether the customer is ready for it or not?

Reply:
no, this should be here, this is what discussed during initial requirements